modules/custom/cu_redirect/cu_redirect.info

```diff
@@ -5,6 +5,7 @@ package = CU-Core
 version = 7.x-1.0
 dependencies[] = ctools
 dependencies[] = redirect
+dependencies[] = match_redirect
```

Comment:
Dependencies must be declared in the profile, core, bundle or hosting.  Those are the only modules that PMM checks.